electrumsv/tests/test_wallet.py

```diff
@@ -532,18 +532,15 @@ def load_wallet(storage: WalletStorage) -> Wallet | None:
 @dataclasses.dataclass
 class State:
     count_of_imported_privkey_account_reencryptions: int = 0
-    count_of_bip39_with_lost_seed_reencryptions: int = 0
-    count_of_bip39_with_saved_seed_reencryptions: int = 0
-    count_of_electrum_seed_per_account_reencryptions: int = 0
-    count_of_account_imported_from_electrum_seed_post_1_4_0: int = 0  # derivation is saved
-    count_of_account_imported_from_bip39_seed_post_1_4_0: int = 0  # derivation is saved
-    count_of_electrum_seed_per_wallet_reencryptions: int = 0
-    count_of_old_electrum_account_reencryptions: int = 0
+    count_of_account_imported_from_bip39_seed_pre_1_3_reencryptions: int = 0
+    count_of_account_imported_from_bip39_seed_1_3_reencryptions: int = 0
+    count_of_account_imported_from_electrum_seed_pre_1_4_reencryptions_reencryptions: int = 0
```

Comment:
I noticed after merging that this looks out of order:
 `reencryptions_reencryptions`
> count_of_account_imported_from_electrum_seed_pre_1_4_reencryptions_reencryptions: int = 0

and this:
> count_of_account_imported_from_bip39_seed_1_4

is missing the `_reencryptions` suffix